pkg/piecestore/rpc/client/utils/utils.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package utils
+
+import (
+	"crypto/md5"
+	"fmt"
+	"io"
+	"os"
+)
+
+// DetermineHash -- Get the hash for a section of data
+func DetermineHash(f *os.File, offset int64, length int64) (string, error) {
+	h := md5.New()
```

Comment:
i don't know that this is how we'll be determining piece hashes going forward in general, but specifically, md5 is very, very, very cryptographically broken. you shouldn't use md5 for anything. sha256 or higher